apps/site/components/Downloads/DownloadReleasesTable/DetailsButton.tsx

```diff
@@ -5,23 +5,19 @@ import type { FC } from 'react';
 import { use } from 'react';
 
 import LinkWithArrow from '#site/components/LinkWithArrow';
-import { ReleaseModalContext } from '#site/providers/releaseModalProvider';
-import type { NodeRelease } from '#site/types';
+import { ModalContext } from '#site/providers/modalProvider';
```

Comment:
Context's should be imported from a hook. Like `useModalContext`

Reply:
And these should be explicitly defined as client-side hooks, so that if imported from server it fails.

Reply:
In the previous implementation, a "useReleaseModalContext" hook wasn't in use, this follows that principle

Reply:
Right, but in general contexts are provided in Hooks. Can we do that please?